pkg/resources/resources.go

```diff
@@ -474,9 +474,9 @@ var assets = func() http.FileSystem {
 		"/traits.yaml": &vfsgen۰CompressedFileInfo{
 			name:             "traits.yaml",
 			modTime:          time.Time{},
-			uncompressedSize: 35740,
+			uncompressedSize: 37013,
```

Comment:
Is is in the scope of that PR?

Reply:
I don't know about this, I run a full build and make images and I'm getting this.

Reply:
I can't track where this could come from, but let's keep it then :)

Reply:
So from a build perspective seems to be in scope.